sdk/core/azure-core/src/main/java/com/azure/core/util/TelemetryAttributes.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
+
+/**
+ * Generic attribute collection applicable to metrics, tracing and logging implementations.
+ * Implementation is capable of handling different attribute types, caching and optimizing the internal representation.
+ */
+public interface TelemetryAttributes {
+}
```

Comment:
Should we have a `get` method to retrieve the value of an attribute key?

Reply:
we don't really need to - implementation does not need to make getter public - it can stay package-private. If at some point we'd need a getter- let's add it then.